lib/streamaggr/streamaggr.go

```diff
@@ -775,9 +777,13 @@ func (a *aggregator) Push(tss []prompbmarshal.TimeSeries, matchIdxs []byte) {
 				// Skip NaN values
 				continue
 			}
+			if sample.Timestamp == 0 {
```

Comment:
Under what conditions could this happen? From the quick check, all ingestion protocols already set timestamp by default.

Reply:
relied on this commit description
https://github.com/VictoriaMetrics/VictoriaMetrics/commit/7acc54025e8ca596147082da29af4a7245026959

Reply:
I see. I checked protocols here https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/lib/protoparser and most of them set timestamp if it is empty. Datadog protocol implementation doesn't do this, but I assume non-zero timestamp is set by datadog agent instead.
I suggest we skip this by now, and add it back once we need it.